File: _ConvertedToCSharp/Library/IRefactoryProvider.cs
//=================================
//Deniz Esen
//Econ Dimension GmbH
//http://www.econdimension.com
//d.esen@econdimension.com
//=================================
// You are free to use, change and
// redistribute this Class or parts
// of it.
// No Support.
//=================================
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Diagnostics;
using System.Drawing;
using System.Windows.Forms;
namespace Econ.NetVert
{
  public interface IRefactoryProvider
  {
    string Name { get; }
    ProviderParserFlags PreAndPostParseFlags { get; }
    string ConvertVBtoCS(string vbSrc);
    string ConvertCStoVB(string csSrc);
  }
}

File: Backup/Library/IRefactoryProvider.vb
'=================================
'Deniz Esen
'Econ Dimension GmbH
'http://www.econdimension.com
'd.esen@econdimension.com
'=================================
' You are free to use, change and
' redistribute this class or parts
' of it.
' No Support.
'=================================

Public Interface IRefactoryProvider

  ReadOnly Property Name() As String

  ReadOnly Property PreAndPostParseFlags() As ProviderParserFlags

  Function ConvertVBtoCS(ByVal vbSrc As String) As String

  Function ConvertCStoVB(ByVal csSrc As String) As String

End Interface
